### ISSUE TITLE
numpy.uint16(...) incorrectly detected as str

### ISSUE THREAD

**Issue by bersbersbers** [PyCQA/pylint#1971]
### Steps to reproduce
1. pylint bug.py
[bug.py.txt](https://github.com/PyCQA/pylint/files/1858058/bug.py.txt)

```
"""Bug"""
import numpy
print(numpy.uint16(0).view())

```

### Current behavior
```
No config file found, using default configuration
************* Module bug
E:  3, 6: Instance of 'str' has no 'view' member (no-member)

----------------------------------------------------------------------
Your code has been rated at -15.00/10 (previous run: -15.00/10, +0.00)

```
### Expected behavior

No pylint error - bug.py runs fine.

### pylint --version output

```
No config file found, using default configuration
pylint 1.8.3,
astroid 1.6.2
Python 3.6.4 (v3.6.4:d48eceb, Dec 19 2017, 06:54:40) [MSC v.1900 64 bit (AMD64)]

```

**Comment by PCManticore** [comment]
This happens because we use strings in the astroid brain tip that we have for numpy: https://github.com/PyCQA/astroid/blob/master/astroid/brain/brain_numpy.py#L153. We should change those to return an object that has the `view()` method and that returns the corresponding type.
CC @hippo91

**Comment by hippo91** [comment]
@PCManticore you are absolutely right. I'll work on it ASAP.
@bersbersbers thanks for the issue.
